src/masternodeman.h

```diff
@@ -137,6 +138,8 @@ class CMasternodeMan
     void Check();
 
     /// Check all Masternodes and remove inactive
+    void CheckAndRemove(CConnman& connman);
+    /// This is dummy overload to be used for dumpong/loading mncache.dat
```

Comment:
Typo: s/pong/ping/ ;)

Also not sure if it makes a lot of sense to have an empty implementation in `.cpp`, could probably just keep it in `.h`
```
void CheckAndRemove(CConnman& connman) {}
```

Reply:
I've pushed updated branch fixing these issues.